README.md

```diff
@@ -1,8 +1,65 @@
+# Country By Country Reporting Frontend
 
-# country-by-country-reporting-frontend
+## Info
 
-This is a placeholder README.md for a new repository
+This service allows a user to upload the correct tax file which further goes for validation and once validated it finally gets uploaded.
```

Comment:
I'd simplify both of these lines to something like:

This service allows a user to upload files as an Agent, Client or Organisation on behalf of an Organisation.

This service has a corresponding back-end service, namely country-by-country-reporting which integrates with HOD i.e DES/ETMP.